src/rpc/server.cpp

```diff
@@ -456,6 +456,11 @@ static inline JSONRPCRequest transformNamedArguments(const JSONRPCRequest& in, c
             hole += 1;
         }
     }
+    auto wallet = argsIn.find("wallet");
+    if (wallet != argsIn.end() && wallet->second->isStr()) {
```

Comment:
`RPC_INVALID_PARAMETER` will be raised when wallet isn't a string because `!argsIn.empty()` will be true below.